docs/manual/scala/guide/cluster/ReadSideSlick.md

```diff
@@ -0,0 +1,98 @@
+# Slick Read-Side support
+
+This page is specifically about Lagom's support for relational database read-sides using JDBC.  Before reading this, you should familiarize yourself with Lagom's general [[read-side support|ReadSide]] and [[relational database read-side support overview|ReadSideRDBMS]].
+
+
+## Configuration
+
+Slick support builds on top of Lagom's support for [[storing persistent entities in a relational database|PersistentEntityRDBMS]]. See that guide for instructions on configuring Lagom to use the correct JDBC driver and database URL.
+
+Next we need to configure the Slick mappings for the read-side model. Note that this example is using the `slick.jdbc.H2Profile`. Make sure you import your profile of choice instead. It's important that you Slick mapping uses the
```

Comment:
I didn't find it. I will remove that sentence. I don't remember what I was planing to say. Probably something related with jdbc profiles, but I don't think we should go too deep on this matters here. Better to stay focused on the core of Slick support in Lagom.